src/spark/cluster.py

```diff
@@ -119,8 +120,18 @@ def create_cluster(
         worker_memory: Memory per worker (e.g., "10GiB")
         master_cores: CPU cores for master
         master_memory: Memory for master (e.g., "10GiB")
+        force: Skip confirmation prompt if True
     """
-    # TODO - inform user that a new cluster will overwrite the existing cluster
+
+    print("WARNING: Creating a new Spark cluster will terminate your existing cluster.")
+    print("All active Spark sessions and computations will be lost.")
+    
+    if not force:
+        confirmation = input("Do you want to proceed? [Y/n]: ").strip().lower() or 'y'
+        if confirmation not in ('y', 'yes'):
+            print("Cluster creation aborted.")
+            return None
```

Comment:
This implies a cluster always exists, otherwise the existence of the cluster should be checked and the function allowed to proceed if there isn't one, regardless of force. Is that right?

Reply:
added a TODO item to check existence of user's clustering.  I think it's not very important at the moment as user's cluster should always exist.